addons/l10n_ar_pos/static/src/overrides/models/pos_store.js

```diff
@@ -6,16 +6,19 @@ import { patch } from "@web/core/utils/patch";
 
 patch(PosStore.prototype, {
     // @Override
-    async _processData(loadedData) {
-        await super._processData(...arguments);
+    async processServerData() {
+        await super.processServerData();
+
         if (this.isArgentineanCompany()) {
-            this.l10n_ar_afip_responsibility_types = loadedData["l10n_ar.afip.responsibility.type"];
-            this.l10n_latam_identification_types = loadedData["l10n_latam.identification.type"];
-            this.consumidorFinalAnonimoId = loadedData["consumidor_final_anonimo_id"];
+            this.consumidorFinalAnonimoId = this.data.custom.consumidor_final_anonimo_id;
+
+            this["l10n_latam.identification.type"] = this.data["l10n_latam.identification.type"];
+            this["l10n_ar.afip.responsibility.type"] =
+                this.data["l10n_ar.afip.responsibility.type"];
```

Comment:
I'm not sure of this change. You didn't replace where `l10n_ar_afip_responsibility_types` and `l10n_latam_identification_types` are accessed.